### ISSUE TITLE
Cannot download Libreoffice documents on iOS App

### ISSUE THREAD

**Issue by eni23** [mattermost/mattermost-mobile#3643]
## Summary

Users cannot download Libreoffice Documents on the iOS Mattermost app. 
Its about the following MIME-Types:

```
application/vnd.oasis.opendocument.text
application/vnd.oasis.opendocument.presentation
application/vnd.oasis.opendocument.spreadsheet	
```

#### Environment Information

- Device Name: Apple iPad Pro 
- OS Version: 12 and 13, both tested
- Mattermost App Version: 1.25.1
- Mattermost Server Version: 5.9 E20

#### Steps to reproduce
Attach a Libreoffice document. It cannot be downloaded or viewed

#### Expected behavior
Document can be downloaded and/or opened with the Collabora iOS app.

#### Possible fixes
Add the following MIME-Types to https://github.com/mattermost/mattermost-mobile/blob/52b26430c118670e1ed050a892e3f36a40fdd7c0/app/components/file_attachment_list/file_attachment_document.js#L26-L36

```
application/vnd.oasis.opendocument.text
application/vnd.oasis.opendocument.presentation
application/vnd.oasis.opendocument.spreadsheet	
```

**Comment by amyblais** [comment]
@eni23 We have a few tickets open for this here:
 - https://mattermost.atlassian.net/browse/MM-20231
 - https://mattermost.atlassian.net/browse/MM-13320

**Comment by eni23** [comment]
Any news on this?

**Comment by eni23** [comment]
The tickets are still in progress. Is there any ETA when this is done?

**Comment by jasonblais** [comment]
Hi @eni23 sorry for the delay in getting back to you.

The tickets were completed a few months ago, and were released in December/2020 (v1.38 of the mobile apps).

I'll close the issue for now. If you see the issue persisting, please ensure your mobile apps are on v1.38 or later, or otherwise reopen this issue and we'll be glad to help you.
